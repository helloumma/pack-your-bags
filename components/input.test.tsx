import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import Input from "./input";

describe("<Input />", () => {
  it("screen matches rendered snapshot", () => {
    const { container } = render(
      <Input
        bag={"test bag"}
        item={"test item"}
        onBagChange={() => null}
        onItemChange={() => null}
        onSubmit={() => null}
      />
    );
    expect(container).toMatchSnapshot();
  });
});
