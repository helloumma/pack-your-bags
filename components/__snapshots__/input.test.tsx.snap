// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> screen matches rendered snapshot 1`] = `
<div>
  <div
    class="w-full flex items-center flex-col justify-center"
  >
    <div
      class="flex  max-w-sm  items-center space-x-2"
    >
      <button
        aria-expanded="false"
        aria-haspopup="menu"
        class="inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50 border border-input bg-background shadow-sm hover:bg-accent hover:text-accent-foreground h-9 px-4 py-2"
        data-state="closed"
        id="radix-:r0:"
        type="button"
      >
        test bag
         
        <svg
          class="h-4 w-4"
          fill="none"
          height="15"
          viewBox="0 0 15 15"
          width="15"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M4 6H11L7.5 10.5L4 6Z"
            fill="currentColor"
          />
        </svg>
      </button>
      <input
        class="flex h-9 w-full rounded-md border border-input bg-transparent px-3 py-1 text-sm shadow-sm transition-colors file:border-0 file:bg-transparent file:text-sm file:font-medium placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50"
        placeholder="Type item to bag..."
        value="test item"
      />
      <button
        class="inline-flex items-center justify-center whitespace-nowrap rounded-md text-sm font-medium transition-colors focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:pointer-events-none disabled:opacity-50 bg-primary text-primary-foreground shadow hover:bg-primary/90 h-9 px-4 py-2"
        type="button"
      >
        Add
      </button>
    </div>
  </div>
</div>
`;
